README.md

```diff
@@ -133,6 +133,9 @@ http://www.github.com/wago/gcc-toolchain-2019.12
     >sudo apt install g++
     >sudo apt install dialog
     >sudo apt install libc6-dev
+    >sudo apt install gettext
+    >sudo apt install pkg-config
+    >sudo apt install libc6-dev
```

Comment:
Please remove this line, since libc6-dev is already provided.

Reply:
fixed. I also added bc, zip and unzip which are needed